scripts/sysman/scheduler.yml

```diff
@@ -22,7 +22,9 @@ etors:
     - name: EXCLUSIVE
       desc: "Any application or context can run indefinitely on the hardware without being preempted or terminated. All pending work for other contexts must wait until the running context completes with no further submitted work."
     - name: COMPUTE_UNIT_DEBUG
-      desc: "This is a special mode that must ben enabled when debugging an application that uses this device e.g. using the Level0 Debug API. It has the effect of disabling any timeouts on workload execution time and will change workload scheduling to ensure debug accuracy."
+      desc:
+        "1.7": "This is a special mode that must ben enabled when debugging an application that uses this device e.g. using the Level0 Debug API. It has the effect of disabling any timeouts on workload execution time and will change workload scheduling to ensure debug accuracy."
```

Comment:
No need to version this description. We can keep it. Only needed is the 1.8 description with the deprecation notice.

If we want to version the old description, it should be 1.0, since this enum is valid since then, but again, it is not needed.

Reply:
so far while deprecating other enums we have added last version up until it is valid. so not sure i follow why it should be "1.0" ?

Reply:
@aravindksg : by versioning with 1.7, you are saying that this text:

"1.7": "This is a special mode that must ben enabled when debugging an application that uses this device e.g. using the Level0 Debug API. It has the effect of disabling any timeouts on workload execution time and will change workload scheduling to ensure debug accuracy."

will appear only with starting at 1.7 version. That means that if we try to build the spec for 1.6 or previous, that text won't appear, and it should.


if we have been doing this that you said:
> so far while deprecating other enums we have added last version up until it is valid

then we have been doing it wrong.

you can confirm this by building the spec for 1.6 with your changes, and confirming whether the text above appears or not.

Reply:
ah ok I see what you mean. on a different PR I think we deprecated statement that was added in 1.7 . fixing this now

Reply:
fixed